include/cvc5/cvc5_proof_rule.h

```diff
@@ -2481,6 +2481,93 @@ enum ENUM(ProofRewriteRule) : uint32_t
    * \endverbatim
    */
   EVALUE(DT_CONS_EQ),
+
+  /**
+   * \verbatim embed:rst:leading-asterisk
+   * **Bitvectors - Unsigned multiplication overflow detection elimination**
+   *
+   * See M.Gok, M.J. Schulte, P.I. Balzola, "Efficient integer multiplication
+   * overflow detection circuits", 2001.
+   * http://ieeexplore.ieee.org/document/987767
+   * \endverbatim
+   */
+  EVALUE(BV_UMULO_ELIMINATE),
+  /**
+   * \verbatim embed:rst:leading-asterisk
+   * **Bitvectors - Signed multiplication overflow detection elimination**
+   *
+   * See M.Gok, M.J. Schulte, P.I. Balzola, "Efficient integer multiplication
+   * overflow detection circuits", 2001.
+   * http://ieeexplore.ieee.org/document/987767
+   * \endverbatim
+   */
+  EVALUE(BV_SMULO_ELIMINATE),
```

Comment:
Since we support Latex, we can copy the formulas needed from the paper. At the very least, we should mention the label/number of the formula used.

Reply:
The formula linked in the paper is very verbose. I don't think we have the space for it